reviewrot/basereview.py

```diff
@@ -82,6 +87,64 @@ def check_request_state(self, created_at,
                 raise ValueError("Invalid duration type: %s" % duration)
         return True
 
+    def _decode_response(self, response):
+        """
+        Remove Gerrit's prefix and convert to JSON.
+        Returns:
+            Converted JSON content
+        Raises:
+            ValueError if the content is not in proper json format.
+        """
+        gerrit_json_prefix = ")]}\'\n"
+        content = response.content.strip()
+        try:
+            if response.encoding:
+                content = content.decode(response.encoding)
+            response.raise_for_status()
```

Comment:
You can probably remove this line because you only call this method if the response was 200. On a side note, if this were to remain, it should be done before attempting to do anything with the content.